website/docusaurus.config.js

```diff
@@ -9,7 +9,7 @@ const config = {
   title: 'Zrok',
   staticDirectories: ['static', '../docs/images'],
   tagline: 'Globally distributed reverse proxy',
-  url: 'https://zrok.io',
+  url: 'https://docs.zrok.io',
```

Comment:
The config prop sets the canonical URL for pages. It must match the actual hosting domain of the site or search engines like Google and Algolia will continue to ignore all child pages of docs.zrok.io.